DirectXTex/DirectXTexFlipRotate.cpp

```diff
@@ -206,7 +251,16 @@ HRESULT DirectX::FlipRotate(
     else
     {
         // Case 2: Source format is not supported by WIC, so we have to convert, flip/rotate, and convert back
-        hr = PerformFlipRotateViaF32(srcImage, flags, *rimage);
```

Comment:
So direct attempts to flip/rotate, resize, or generate mips on a 16834 x 16834 ``DXGI_FORMAT_R32B32G32A32_FLOAT`` image using WIC is going to continue to fail even with these changes as WIC just can't handle that size. The purpose of this fix is to make *other* formats work that don't have a direct DXGI-format equivalent in WIC, which when converted to ``DXGI_FORMAT_R32B32G32A32_FLOAT`` exceed a 32-bit size.